corehq/motech/dhis2/management/commands/populate_couch_datasetmap.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management import BaseCommand
+
+from ...models import DataSetMap, DataValueMap, SQLDataSetMap
+
+
+class Command(BaseCommand):
```

Comment:
This management command creates Couch instances for new SQLDataSetMaps that have been created since deploy/migration. (If no new SQLDataSetMaps have been created since deploy, this command is not necessary for rollback.)

It does not update old Couch instances with changes made in migrated SQLDataSetMaps. We could support that if we added a "last_modified" datetime to the SQL model.

Alternatively, we could avoid the need for this command by duplicating changes across Couch and SQL.

I felt these approaches seemed unnecessary for models that don't have a lot of instances and don't get a lot of churn. I'm open to your thoughts about this.